todo_app/app.py

```diff
@@ -1,11 +1,20 @@
-from flask import Flask
+from flask import Flask, redirect, render_template, request, url_for
 
+from todo_app.data.session_items import add_item, get_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
+# Display list of items
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items=items)
+
+# Create new items
+@app.route('/action', methods=['POST', 'GET'])
```

Comment:
Is there any reason to allow GET methods here? You only expect the user to hit the address via POST, so I'd be tempted to block GET requests - a clear error message when something unusual is happening is often better than just allowing it in case